main.dart

```diff
@@ -69,31 +74,39 @@ Future<void> createDatabase() async {
 
 Future<void> listDatabases() async {
   print('Running List Databases API');
-  final databases = Databases(client, databaseId: databaseId);
+  final databases = Databases(client);
   try {
-    final db =
-        await databases.list();
+    final db = await databases.list();
     print(db.toMap());
   } on AppwriteException catch (e) {
     print(e.message);
   }
 }
 
 Future<void> createCollection() async {
-  final database = Databases(client, databaseId: databaseId);
+  final database = Databases(client);
   print('Running create collection API');
   try {
     final res = await database.createCollection(
-      collectionId: "movies",
-      permission: 'document',
+      databaseId: databaseId,
+      collectionId: ID.custom("movies"),
```

Comment:
If we use unique here we can run the playground multiple times without issue

```suggestion
      collectionId: ID.unique(),
```

Reply:
Updated!